docs/Installation.md

```diff
@@ -52,6 +52,12 @@ for Zonemaster::Backend, see the [declaration of prerequisites].
 > **Note:** In addition to the normal dependencies, the post-installation
 > smoke test instruction assumes that you have curl installed.
 
+Optionally install jq (only needed for the post-installation smoke test):
+
+```sh
+sudo yum install jq
+```
+

```

Comment:
These six lines (CentOS specific) do not belong to the general Prerequisites section. Suggested to be replaced by the lines below.

Reply:
Right. I'm moving them.